node/resolvers/search/index.ts

```diff
@@ -245,6 +243,35 @@ export const queries = {
     return search.products(args)
   },
 
+  productsNoSimulations: async (_: any, args: SearchArgs, ctx: Context) => {
+    const {
+      clients: { search },
+    } = ctx
+    const queryTerm = args.query
+    if (queryTerm == null || test(/[?&[\]=]/, queryTerm)) {
+      throw new UserInputError(
+        `The query term contains invalid characters. query=${queryTerm}`
+      )
+    }
+
+    if (args.to && args.to > 2500) {
+      throw new UserInputError(
+        `The maximum value allowed for the 'to' argument is 2500`
+      )
+    }
+
+    const products = await search.products(args)
```

Comment:
Shouldn't you add `skipSimulation = true` to the args

Reply:
nice catch